public/components/BetterComboBox/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { EuiComboBox, EuiComboBoxProps } from "@elastic/eui";
+
+export default class BetterComboBox<T> extends EuiComboBox<T> {
```

Comment:
The name `BetterComboBox` doesn't describe what enhacement is made, can we use a name that indicates the function of this extension?

Reply:
similarly for `BetterTooltip`

Reply:
Fixed on https://github.com/opensearch-project/index-management-dashboards-plugin/commit/70dd00514c00fd9789bef132ebf981ef9eda0ffd